### PR TITLE
ripgrep: update to 14.1.0

### PR DESCRIPTION
**Pull request by herbygillot**
~~This probably won't build until https://github.com/macports/macports-ports/pull/21985 gets merged.~~

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint --nitpick`?
- [ ] tried existing tests with `sudo port test`?
- [ ] tried a full install with `sudo port -vst install`?
- [ ] tested basic functionality of all binary files?
- [ ] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

**Comment by herbygillot** [comment]
This is ready for review, @raimue.

File: textproc/ripgrep/Portfile [hunk 1]
```diff
@@ -4,102 +4,116 @@ PortSystem          1.0
...
+                    size    584473
 
 depends_build-append \
                     port:asciidoc \
                     port:docbook-xsl-nons
 
+set ripgrep_bin     ${worksrcpath}/target/[cargo.rust_platform]/release/rg
+
+post-build {
+
```
Comment (raimue) [inline textproc/ripgrep/Portfile:83]:
```suggestion
```

**Review by raimue — COMMENTED** [review]
Looks good to me overall. Thanks for stepping up as maintainer! :+1:

**Comment by herbygillot** [comment]
Thanks @raimue

**Pull request merged**